website_rma/views/request_rma_form.xml

```diff
@@ -0,0 +1,223 @@
+                                                    t-foreach="request.env['rma.operation'].sudo().search([])"
+                                                    t-as="o"
+                                                >
+                                                    <option t-att-value="o.id">
+                                                        <t t-esc="o.name" />
+                                                    </option>
+                                                </t>
+                                            </select>
+                                        </div>
+                                    </div>
+                                    <div
+                                        class="form-group row form-field o_website_form_required_custom"
+                                    >
+                                        <label
+                                            class="col-lg-3 col-md-4 col-form-label"
+                                            for="description"
+                                        >Description</label>
+                                        <div class="col-lg-7 col-md-8">
+                                            <textarea
+                                                class="form-control o_website_form_input"
+                                                name="description"
+                                                required=""
+                                            >
+                                                <t
+                                                    t-esc="request.params.get('description', '')"
+                                                />
+                                            </textarea>
+                                        </div>
+                                    </div>
+                                    <div class="form-group row">
+                                        <div
+                                            class="offset-lg-3 offset-md-4 col-md-8 col-lg-7"
+                                        >
+                                            <a
+                                                href="#"
+                                                role="button"
+                                                class="btn btn-primary btn-lg o_website_form_send"
+                                            >Request</a>
+                                            <span id="o_website_form_result" />
+                                        </div>
+                                    </div>
+                                </form>
+                            </div>
+                            <div
+                                class="oe_structure mt-2"
+                                id="oe_structure_website_rma_form_request_2"
+                            />
+                        </div>
+                        <t t-call="website_rma.company_description" />
```

Comment:
Well, it's useless to create this additional view with demo data. Odoo's approach now is that the user simply edits the page in the website and thus a custom website is created. So for me, this block can simply go away and let the user custom the page as he please